manifests/repo/redhat.pp

```diff
@@ -7,12 +7,13 @@
 #
 
 class php::repo::redhat (
-  String[1] $yum_repo = 'remi_php56',
```

Comment:
I can rework the code to keep the original param name if that makes more sense to keep it backwards compatible.

Reply:
Only keep it unchanged if it makes sense: breaking backwards compatibility is not something that should stop you :wink:   We (voxpupuli team) just want not to forget about this kind of things so that end-users can trust our minor updates to not break their setup!

Reply:
I think it would be better to keep the backwards compatibility - I wasn't really thinking in those terms, and there really isn't a need to change it other than cosmetics.   I pushed the change.